src/platforms/apple/common/enriching-events/user-feedback.mdx

```diff
@@ -0,0 +1,29 @@
+---
+title: "User Feedback"
+sidebar_order: 105
+
+description: "Learn more about the user feedback API, which gives you the possibility to collect user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues."
+---
+
+The user feedback API gives you the possibility to collect user information when an event occurs. Sentry pairs the feedback with the original event, giving you additional insight into issues.
+Sentry needs the `eventId` to be able to associate the user feedback to the corresponding event. To get the `eventId` you for example [`beforeSend`](/platforms/apple/guides/ios/configuration/options/#before-send) or use the return value of the methods capturing an event.
```

Comment:
Can anyone help me out on how to add a proper link here?

Reply:
@philipphofmann the format is right, but its probably the ``, try doing:

[beforeSend](/platforms/apple/guides/ios/configuration/options/#before-send)

`beforeSend` will be highlighted automatically, you don't need to do yourself

Reply:
But the link is always for iOS. The link should keep you on the platform as iOS, tvOS, etc. @marandaneto

Reply:
then you can just use `platforms/apple/configuration/filtering/#using-beforesend` or `platforms/apple/configuration/options/#before-send`

just omit `guides/ios` in the URL

Reply:
But then if you are on iOS, you leave iOS and end up on Apple. I would like to stay on Apple, when I'm on Apple and stay on iOS when I'm on iOS. @marandaneto

Reply:
tbh I don't think it'd be possible because of the way the docs are structured on the apple page, the `guides` creates more compressibility for this I guess.
maybe @getsentry/docs

Reply:
@marandaneto you can do it like this `<PlatformLink to="/configuration/options/#before-send">beforeSend</PlatformLink>`